testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/webauthn/AppPage.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.testsuite.pages.webauthn;
+
+import org.keycloak.OAuth2Constants;
+import org.keycloak.protocol.oidc.OIDCLoginProtocolService;
+import org.keycloak.testsuite.pages.AbstractPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import javax.ws.rs.core.UriBuilder;
+
+/**
+ * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
+ */
+public class AppPage extends AbstractPage {
```

Comment:
This matter is induced by using ChromeDriver not managed by arquillian/drone on RegisterAndLoginTest.java .

As you mentioned testsuite/integration-arquillian/tests/base/pom.xml below, this can be resolved by using ChromeDriver managed by arquillian/drone .

```java
    @FindBy(id = "account")
    private WebElement accountLink;
```
This accountLink is bound to the web element by WebDriver managed by arquillian/drone .
Therefore, it can not be used by RegisterAndLoginTest because it uses ChromeDriver not managed by arquillian/drone.

``` java
    public void openAccount() {
        driver.findElement(By.id("account")).click();
        //accountLink.click();
    }
```
Therefore, I've modified the method above.

Reply:
Yes, +1 for new class and override only openAccount(). Hope we can manage to have ChromeDriver managed by arquillian/drone, but that's not a blocker for this PR and can be a done as a follow-up task IMO.